### DISCUSSION TITLE
Multiple Roller Shutters Moving Simultaneously (ESP 8266)

### DISCUSSION THREAD

**Discussion by hayta78** [arendst/Tasmota#21005] [Q&A]
Hi there,

Newbee to Tasmota. I've attached a 4 relay serial board to Tasmota (on ESP 8266) that runs two roller shutters. While one roller shutter is moving if I send a command to the other one, the moving one stops other one start moving. Is there a way to make them operating simultaneously? I've done a quick check on the documentation and found no info.

Thanks,
Kerem

**Comment by stefanbode** [comment]
This should work. You should have an interlock on relay 1,2 and one on 3,4. All this in detailed explained in the documentation
